docs/docs/testing/e2e/app-template.md

```diff
@@ -0,0 +1,46 @@
+---
+title: App Template
+tags:
+  - testing
+  - e2e
+  - template
+  - infrastructure
+---
+
+## Overview
+
+Here you can read about what content schemas the test instance has & the API customisations we've built (incl. why we built them).
+
+## Content Schemas
+
+:::note
+There's no content yet!
+:::
+
+## API Customisations
+
+### Database
+
+found at `template/src/api/database`
+
+#### Usage
+
+```ts
+import { test } from '@playwright/test';
+
+test.beforeEach(async ({ page }) => {
+  await page.request.fetch('http://localhost:1337/api/database-dump', {
+    method: 'POST',
+  });
+});
```

Comment:
Isn't that outdated now that we are using DEITS directly? In any case the URL should be `/api/database` :)

Reply:
I'd like to keep it just incase, it's very useful! i've amended the docs, looking at the route it should be `database/dump` 👍🏼